_includes/commands/table-cmds.md

```diff
@@ -0,0 +1,42 @@
+
+| Command                 | Description                                                                |
+| ----------------------- | -------------------------------------------------------------------------- |
+| mceTableSplitCells      | Splits the current merge table cell.                                       |
+| mceTableMergeCells      | Merges the selected cells or opens a dialog if there is no cell selection. |
```

Comment:
This appears to be incorrect, as I tested it out and there's no dialog that opens. If there's only a single cell selected, or none at all then it just does nothing.

Reply:
deleted incorrect bit